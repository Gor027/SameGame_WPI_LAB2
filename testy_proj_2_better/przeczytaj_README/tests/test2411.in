55  5  5  o555o 55o 555 o o5
oo      oo 55oo5 5o5o   5o5   5oo oo555oo   o 5o o55o 5oo o5 5 o 5o 5 5 o o oo5
 5o55o5ooo 5ooo5o5   55o  5o55o
o55
55ooo5o5o 55 ooo o5oo oo55oo 5o o o   oo  5o5o5 5 o  5   
o     o5 o o o 5o5      5  5    o5oo555 
  o5oo  5o5 555o5o5 55oo oo o oo  o5555oo55ooo55 55 oo5  5o55   o55o55 o5
5oo5 55  o555 oo o5ooooo5oo55o55555o5oo5 oo 5oo 5o5555oo  o oo 5o   5o5
 
5 5oo 5  o5o o5 oo55o 5o oo  5oo 55o 55oo 5 o5  55
 5o   5 o555555o ooo o5  5  o 5 5oo 5 oo55 555o
