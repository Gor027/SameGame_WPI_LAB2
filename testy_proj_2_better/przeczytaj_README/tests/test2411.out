                 5
           5 5o  5
  o      o 5 oo5 5o   5   o   5   o   5
 55   5  oo55oo5 oo 5 5  o5   o   o   oo    o
5oo55 o oo555ooo5555o5ooo555  o o oo555o    5o5
o5oo5oo5oo5ooo55oo5oo5oo5ooo55ooo5oo555oo 5 oo5  o5
o5o5o55ooo5555oooo5o55oo5oooo55oo55555o5o55o5o5 55o o55   o        5
5555oo555o5o5o5o5o55oo5o5oo55oo5555o5o5oooooo5oo5o55oo5o  o555oo5o 555 o
ooooo555o555555ooooooo5o55o5o55o5ooo55oo555555o5555555ooo5o5oo5o55o5o5o5ooo5
